Sources/TuistCore/Models/Project.swift

```diff
@@ -67,23 +63,21 @@ public class Project: Equatable, CustomStringConvertible {
     ///                      *(Those won't be included in any build phases)*
     public init(path: AbsolutePath,
```

Comment:
I removed the defaults to prevent `with` methods from creating copies with missing attributes.